circleci.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -uexo pipefail
+
+HOST_DMD_VER=2.068.2 # same as in dmd/src/posix.mak
+CURL_USER_AGENT="CirleCI $(curl --version | head -n 1)"
+N=2
+
+case $CIRCLE_NODE_INDEX in
+    0) MODEL=64 ;;
+    1) MODEL=32 ;;
+esac
+
+install_deps() {
+    if [ $MODEL -eq 32 ]; then
+        sudo apt-get update
+        sudo apt-get install g++-multilib
+    fi
+
+    for i in {0..4}; do
+        if curl -fsS -A "$CURL_USER_AGENT" --max-time 5 https://dlang.org/install.sh -O; then
+            break
+        elif [ $i -ge 4 ]; then
+            sleep $((1 << $i))
+        else
+            echo 'Failed to download install script' 1>&2
+            exit 1
+        fi
+    done
+
+    source "$(CURL_USER_AGENT=\"$CURL_USER_AGENT\" bash install.sh dmd-$HOST_DMD_VER --activate)"
+    $DC --version
+    env
+}
+
+coverage() {
+    if [ -n ${CIRCLE_PR_NUMBER:-} ]; then
+        local base_branch=$(curl -fsSL https://api.github.com/repos/dlang/dmd/pulls/$CIRCLE_PR_NUMBER | jq -r '.base.ref')
+    else
+        local base_branch=$CIRCLE_BRANCH
+    fi
```

Comment:
I hope we don't , otherwise we can try 1) direct [authenticated access](https://developer.github.com/v3/#oauth2-keysecret) or 2) rolling our own, small API cache